src/Control/Exception/Safe.hs

```diff
@@ -439,6 +425,29 @@ bracketOnError before after thing = C.mask $ \restore -> do
 bracketOnError_ :: C.MonadMask m => m a -> m b -> m c -> m c
 bracketOnError_ before after thing = bracketOnError before (const after) (const thing)
 
+-- | Async safe version of 'E.bracket' with access to the exception in the
+-- cleanup action.
+--
+-- @since 0.1.0.0
```

Comment:
This version number isn't correct. Can you bump the version number in the cabal file, add a changelog entry, and fix the version number here? This should be a minor version bump.